app/src/main/java/space/taran/arknavigator/utils/FileUtils.kt

```diff
@@ -94,8 +94,13 @@ fun findLongestCommonPrefix(paths: List<Path>): Path {
     return tailrec(ROOT_PATH, paths).first
 }
 
-fun extension(path: Path): String =
-    path.fileName.toString()
-        .split('.')
-        .drop(1)
-        .last()
\ No newline at end of file
+fun extension(path: Path): String {
+    val pathString = path.fileName.toString()
+    return if (pathString.contains(".") &&
+        !pathString.split(".").drop(1).isNullOrEmpty())
+        pathString
+            .split('.')
```

Comment:
I think we don't need several calls to searching functions.
Maybe something like this? https://tinyurl.com/we3wrzm2